packages/web/integrations/ethereum/types.ts

```diff
@@ -24,23 +24,29 @@ export interface EthWallet
 
 export type SendFn = Pick<EthWallet, "send">["send"];
 
-/** Ethereum chains: https://docs.metamask.io/guide/ethereum-provider.html#chain-ids */
 export const ChainNames: { [chainId: string]: string } = {
-  "0x1": "Ethereum",
+  /** Ethereum chains: https://docs.metamask.io/guide/ethereum-provider.html#chain-ids */
+  "0x1": "Ethereum Main Network",
```

Comment:
Why "Ethereum" -> "Ethereum Main Network"?

If these are different from `SourceChain` type, things will break. We reverse-map (right -> left, or name -> hex ID) to cache EVM chain IDs in the browser ([see](https://github.com/osmosis-labs/osmosis-frontend/blob/JP69/stage-erc-axl-xfer/packages/web/integrations/axelar/transfer.tsx#L78))

Reply:
The reason for the change is because of the name for it shown here: https://docs.metamask.io/guide/ethereum-provider.html#chain-ids
The corresponding change was also made in the map: EthClientChainIds_AxelarChainIdsMap, corresponding the new EthClientChainId of Ethereum Main Network to the Axelar-accepted value of "Ethereum", so I don't think it will break. Upon testing, it still prompts my Metamask to switch to Ethereum Mainnet.
I can easily change it back to just 'Ethereum' if you like.